packages/mux-uploader/src/progress.ts

```diff
@@ -0,0 +1,167 @@
+import { globalThis, document } from 'shared-polyfills';
+
+const template = document.createElement('template');
+const TYPES = {
```

Comment:
nitpick(blocking): Use the `ProgressTypes` lookup object defined in `constants` and update references in this module accordingly.